sys/fs/cd9660/cd9660_node.c

```diff
@@ -146,8 +146,22 @@ cd9660_defattr(struct iso_directory_record *isodir, struct iso_node *inop,
 	}
 	if (!ap) {
 		inop->inode.iso_mode |= S_IRUSR|S_IXUSR|S_IRGRP|S_IXGRP|S_IROTH|S_IXOTH;
-		inop->inode.iso_uid = (uid_t)0;
-		inop->inode.iso_gid = (gid_t)0;
+		/*
```

Comment:
The changes in this function should be a separate commit with its own log message.

Reply:
Fixed.